src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -97,7 +97,7 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                                 ? new KeyValuePair<string, object>("OriginalFormat (a.k.a Body)", listKvp[i].Value)
                                 : listKvp[i];
 
-                            if (ConsoleTagTransformer.Instance.TryTransformTag(listKvp[i], out var result))
+                            if (ConsoleTagTransformer.Instance.TryTransformTag(valueToTransform, out var result))
```

Comment:
This looks unrelated to the sample. I was going to update the PR title but I thought I would double-check you intended to include it first.